styleguide-app/View.elm

```diff
@@ -18,7 +20,18 @@ import Update exposing (..)
 
 view : Model -> Document Msg
 view model =
-    { title = "Style Guide", body = [ view_ model |> Html.toUnstyled ] }
+    { title = "Style Guide"
+    , body =
+        [ view_ model |> Html.toUnstyled
+        , RootHtml.node "link"
+            [ Html.Attributes.href "https://fonts.googleapis.com/css?family=Muli:400,400i,600,600i,700,700i,800,800i,900,900i"
+            , Html.Attributes.rel "stylesheet"
+            ]
+            []
+        , RootHtml.div [ Html.Attributes.id "svg-target" ] []
+        , RootHtml.node "SCRIPT" [ Html.Attributes.src "bundle.js" ] []
```

Comment:
It is because we're using Browser.application, yes. And we are mucking with url. But I'm not sure how vital it actually is that we use application or not. Looking at the history, it looks like maybe @brookeangel has context?

Reply:
if the goal is to get it to work with the `docs` stuff in the other PR, it's reasonably easy to port the previous approach with very little `Makefile` magic 😃

Reply:
sorry, to be more specific: if moving the script into the Elm app is in aid of that goal. If it's necessary to get something else to work, my comment is irrelevant!

Reply:
😅 sorry, "port the previous approach " -- which previous approach? I'm not quite following

Reply:
Do you mean context on why there is a `bundle.js`? (Not sure what exactly you are referring to here)

Reply:
lemme use more words: if you consolidated all the files to Elm nodes in order to get it to output to `docs`, then we can modify the `Makefile` to copy `index.html` instead. That would avoid having to do `RootHtml.node "SCRIPT"`. If this line is due to some other thing and I am not getting it yet, then ignore me. 😄

Reply:
Brooke, sorry -- I was curious if you had context on when we started using Browser.application instead of.. Browser.document? element? whatever we were using when we set up the styleguide application initially.

When we first set up the application, the styleguide readme said to do:

```
elm-make --output=elm.js
open index.html
```
We were explicitly outputting to a js file because there are/were assets that we care about in the styleguide application (svgs, the clipboard.js dealio, the fonts) that we were loading in our hand-written index.html file.

However, at some point, we started using make and Browser.application. At this point, the bundle.js file stopped actually being loaded in the styleguide. You'll noticed that if you run the styleguide locally on master, you will have missing icons and missing fonts and the copy button won't work.

This PR is only intended to get all the assets -- fonts and svgs and scripts -- back up and running. (In general, I'm trying to do prs related to icons right now, and it's deeply unhelpful to not be able to **see** the icons while developing).

Why does this pr drop the original hand-written index.html file? that bit is kind of a combo of reasons:
1. we must have a server running to use Browser.application applications
2. I want the styleguide to be statically served at gh-pages
3. it didn't occur to me to just like.. run a server :P

Reply:
> Brooke, sorry -- I was curious if you had context on when we started using Browser.application instead of.. Browser.document? element? whatever we were using when we set up the styleguide application initially.

Er... probably because this was the first elm 0.19-ify thing I did 😳 feel free to change it!

Reply:
I looked into it more -- it definitely makes sense that it uses `application`!  How we do SPAs elsewhere (like Nri.Spa and such) is much more complex. I think it's worth sticking with `application`